index.html

```diff
@@ -16,51 +16,75 @@
   <body>
     <div id="gradient"></div>
     <!-- NAV SECTION -->
-    <section id="top-nav" class="container">
-      <button id="hamburgerNav">
-        <span id="hamburger1"></span>
-        <span id="hamburger2"></span>
-        <span id="hamburger3"></span>
-      </button>
-      <a href="#hero">
-        <img id="logo" src="assets/WT_logo_white.svg" alt="home" />
-      </a>
-      <nav id="site-nav">
-        <ul>
-          <li><a href="#hero" class="-active">Home</a></li>
-          <li><a href="#about">The Mission</a></li>
-          <li><a href="#schedule">Schedule</a></li>
-          <li><a href="#rules">Rules</a></li>
-          <li><a href="#faqs">FAQs</a></li>
-          <li><a href="#registration">Registration</a></li>
-        </ul>
-      </nav>
-      <nav class="social-nav">
-        <ul>
-          <li>
-            <a href="https://www.facebook.com/willowtreeapps/" target="_blank"
-              ><img src="./assets/facebook.svg" alt="facebook"
-            /></a>
-          </li>
-          <li>
-            <a href="https://twitter.com/willowtreeapps/" target="_blank"
-              ><img src="./assets/twitter.svg" alt="twitter"
-            /></a>
-          </li>
-          <li>
-            <a href="https://www.instagram.com/willowtreeapps/" target="_blank"
-              ><img src="./assets/instagram.svg" alt="instagram"
-            /></a>
-          </li>
-          <li>
-            <a
-              href="https://www.linkedin.com/company/willowtreeapps"
-              target="_blank"
-              ><img src="./assets/linkedin.svg" alt="linkedin"
-            /></a>
-          </li>
-        </ul>
-      </nav>
+    <section id="top-nav">
```

Comment:
That's a good question, I'm not sure... Can you check with Mohammad? This might be complicated because if we take away the active styling from the active link, we would need a way to remember what link is active so that it regains the active styling whenever the nav links lose focus, if that makes sense? I think a simpler solution could maybe be making the hover/focus styles different than the active link styles so both can exist at the same time while not being potentially confusing for a user? but I'll leave that up to Mohammad. :-)